tests/wicked/startandstop_sut.pm

```diff
@@ -109,6 +109,39 @@ sub run {
     }
     mutex_create("test_5_ready");
 
+    $self->before_scenario('Test 7', 'Bridge - ifdown, create new config, ifreload, ifdown, ifup', $iface);
+    $config = '/etc/sysconfig/network/ifcfg-br0';
+    $self->get_from_data('wicked/ifcfg/br0',    $config);
+    $self->get_from_data('wicked/ifcfg/dummy0', $dummy);
+    $self->setup_bridge($config, $dummy, 'ifup');
+    assert_script_run('wicked ifdown --timeout infinite br0');
+    assert_script_run('wicked ifdown --timeout infinite dummy0');
+    $results{7} = sub {
```

Comment:
good idea ! it's better than endless if/else branches ! 
But will be dropped anyway if we will merge refactoring which @jlausuch  is doing :)

Reply:
true :)